src/transpiler.js

```diff
@@ -95,6 +97,14 @@ function getFamilyName({ names = undefined }) {
   return names && names[0] && names[0].familyName
 }
 
+function getCompany({ organizations = undefined }) {
+  return organizations && organizations[0].name
+}
+
+function getNote({ userDefined = undefined }) {
+  return (userDefined && userDefined.find(ud => ud.key === 'note').value) || ''
+}
```

Comment:
`getCompany` returns `undefined, but `getNote` returns `''`. Is is deliberate?

Reply:
right, changed it to `undefined`, it's more consistent